### ISSUE TITLE
Notifications/Activities Dropdown Panel Feature Request

### ISSUE THREAD

**Issue by whizkidwwe1217** [vmware/clarity#2350]
**Select one ...** (check one with "x")

```
[ ] bug
[x] feature request
[ ] enhancement
```

### Expected behavior
This is usually used in headers where there's a notifications icon that when clicked, it shows some recent information collected by the web app. This is very useful in displaying recent notifications such as when a download is completed or when an activity arrives and needs an attention from the user. Also useful when you have an internal mailing system, a helpdesk, or a project tasks tracking system.

Actual use case: I'm gonna use this for my payroll system where immediate superiors can quickly see recent filings of leaves, change shifts, etc. in the header of the web app without navigating to another page.

[Samples](https://medium.com/ux-power-tools/design-an-adaptive-notification-dropdown-in-sketch-7cf406d70fe8):
![image](https://user-images.githubusercontent.com/1977391/41266563-0c36ae00-6e2a-11e8-8172-f99af234778a.png)

### Actual behavior
I know that this will need a lot work and I'm not sure if you consider including this type of component in Clarity UI but if you can provide just a blank panel that behaves like a notification dropwdown and can be placed in the header, then that would be a great help for me. Thanks :)

### Reproduction of behavior
I created my own implementation of this component in StackBlitz, [https://clarity-light-theme-v11-brv6wm.stackblitz.io/](https://clarity-light-theme-v11-brv6wm.stackblitz.io/) but I'm neither a front-end developer nor a UX designer so it's not that good :) I have no idea how all the things work such as positioning of the pop up or when to trigger the popup, etc. It would be much appreciated if Clarity have this kind of component. 
![image](https://user-images.githubusercontent.com/1977391/41299219-95f06328-6e95-11e8-9a0c-de9b8a7ed9e8.png)

* Include a link to the reproduction scenario you created by forking one of the Clarity Plunker Templates:
  
  
  
  

### Environment details

* **Angular version:** 4.x.x

* **Clarity version:**

* **OS and version:**

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by pumano** [comment]
Looks like layout can be used from "alerts" widget from here: https://dribbble.com/shots/4577060-Dashboard-exploration/attachments/1034895

**Comment by colinreedmiller** [comment]
This is an excellent request and one we should implement. The current concern is that we have a number of different alert / notification / toast / snackbar components which need to be systematically unified re: visual design, use case function/purpose, and interaction patterns. When we address this, adding a notification drawer / badged icon function makes perfect sense, but until then it will be on hold, essentially blocked by the larger unification effort.

**Comment by gnomeontherun** [comment]
This has been linked to the Toasts/Notifications #365 request and should be considered as part of the overall notifications pattern we've been developing. I'm going to close this separate issue so we can streamline our view of requests.
